### PR TITLE
Update sass and docker

### LINKED ISSUE #1793: Current version of sass preventing ability to run on node lts
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description

The docker image the dashboard is built on isn't lts. The reason for this is the sass is too outdated and can't run on node lts

### Steps to reproduce

Try to build the dashboard on node lts

### Actual Outcome

Build will fail

### Expected Outcome

Dashboard should always build on lts.

### Environment

Dashboard
- Parse Dashboard version: `master`
- Browser (Safari, Chrome, Firefox, Edge, etc.): `Safari`
- Browser version: `N/A`

Server
- Parse Server version: `4.10.3`
- Operating system: `Docker linux`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `local`

Database
- System (MongoDB or Postgres): `Postgres`
- Database version: `13`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `local`

### Logs

https://github.com/netreconlab/parse-dashboard/runs/3545759058?check_suite_focus=true#step:7:378

### PR DESCRIPTION
**Pull request by cbaker6**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Issue Description

Related issue: #1793 Close #1793 

### Approach

Update sass to versions before breaking changes are introduced. This allows the dashboard to be built on latest node lts. It also aligns the Dashboard with the same base image as the parse-server [Dockerfile](https://github.com/parse-community/parse-server/blob/7807d6343d822d595ed99956bfc6510d8cc5496f/Dockerfile#L2).

Note: to update to the latest version of sass, other parts of the dashboard will need updating which I will leave to someone else more knowledgeable.

For reference of working images:

- my [branch](https://github.com/netreconlab/parse-dashboard/tree/dockerFix)
- the images it produced, `latest`, and `2.2.2`, https://registry.hub.docker.com/repository/docker/netreconlab/parse-dashboard/tags?page=1&ordering=last_updated

### TODOs before merging

- [x] Add entry to changelog
- [x] Update package-lock.json

### CODE REVIEW

**Comment by mtrezza** [comment]
You made it pass! That means alpine is not running on Node 16 I guess, because that would require [node-sass 6.0](https://github.com/sass/node-sass/releases/tag/v6.0.0)

**Comment by cbaker6** [comment]
> You made it pass! That means alpine-lts is not running on Node 16 I guess, because that would probably require [node-sass 6.0](https://github.com/sass/node-sass/releases/tag/v6.0.0). You can test that if you merge master and enable the node 16 check in `ci.yaml`.
> 
> https://github.com/parse-community/parse-dashboard/blob/a6372567aed47670e17b48a8c028207eb0f53825/.github/workflows/ci.yml#L133-L135

Probably so, I tried to build it with node 16 on my local machine and it wouldn’t.

File: package.json [hunk 1]
```diff
@@ -135,7 +135,7 @@
     "parse-dashboard": "./bin/parse-dashboard"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=12.20.0"
```
Comment (mtrezza) [inline package.json:138]: Can you revert this change?

**Comment by mtrezza** [comment]
I have no permission to edit the files, so I just made a comment for the node version change, and if you merge master into this, it should be good to merge.
